src/main/java/calculator/service/SeparatorServiceImpl.java

```diff
@@ -0,0 +1,58 @@
+package calculator.service;
+
+import calculator.config.SeparatorConstants;
+import calculator.model.Separator;
+import calculator.repository.SeparatorRepository;
+import java.util.Set;
+
+public class SeparatorServiceImpl implements SeparatorService{
+    private final SeparatorRepository separatorRepository;
+
+    public SeparatorServiceImpl(SeparatorRepository separatorRepository) {
+        this.separatorRepository = separatorRepository;
+    }
+
+    @Override
+    public void join(Separator separator) {
+        if (!have(separator)) {
```

Comment:
```suggestion
        if (!isExist(separator)) {
```
위와 같은 메소드명이 더 적절한 의미를 전달할 수 있을 것 같습니다.